pytest/lib/mocknet.py

```diff
@@ -440,10 +440,9 @@ def compress_and_upload(nodes, src_filename, dst_filename):
 # We assume that the nodes already have the .near directory with the files
 # node_key.json, validator_key.json and config.json.
 def create_and_upload_genesis(validator_nodes,
```

Comment:
FYI, if you want to require those to be passed as keyword arguments you can add `*` as argument in the definition, e.g.:

    def create_and_upload_genesis(*,
                                  validator_nodes,
                                  chain_id,
                                  …):